turbo.json

```diff
@@ -33,7 +31,7 @@
       "dependsOn": []
     },
     "//#check:fmt": {
-      "dependsOn": ["//#check:prettier", "//#check:toml", "//#lint"]
+      "dependsOn": ["//#check:prettier", "//#check:toml", "lint"]
```

Comment:
We don't even have a `//#check:fmt` script. This is a fake node.

Reply:
it's invoked here: https://github.com/vercel/turbo/blob/f46865809a783d04cb2df756b44cd0124abbeb84/.github/workflows/test.yml#L860-L861. 

I'm guessing it's for the sole purpose of invoking these dependencies